pkg/config/config.go

```diff
@@ -100,6 +100,9 @@ type Config struct {
 	EnableJitterBuffer     bool    `yaml:"enable_jitter_buffer"`
 	EnableJitterBufferProb float64 `yaml:"enable_jitter_buffer_prob"`
 
+	// PreventTransferForInternalProvider prevents SIP transfers when the provider is internal and config is set.
+	PreventTransferForInternalProvider bool `yaml:"prevent_transfer_for_internal_provider"`
```

Comment:
Is it something specific to a particular provider? Can IO send this setting instead as a part of `ProviderInfo` (e.g. `ProviderInfo.PreventTransfers`)?